packages/ember-application/lib/system/application.js

```diff
@@ -37,7 +37,7 @@ import LegacyEachView from 'ember-views/views/legacy_each_view';
 import LinkToComponent from 'ember-routing-views/views/link';
 import RoutingService from 'ember-routing/services/routing';
 import ContainerDebugAdapter from 'ember-extension-support/container_debug_adapter';
-
+import { _loaded } from 'ember-runtime/system/lazy_load';
```

Comment:
at first glance this does seem strange `import { _blah }` I looked at the definition and don't know whether this is private just am assuming that the underscore prefix means private.